Plugin/ProductList/Crosssell.php

```diff
@@ -1,4 +1,10 @@
 <?php
+declare(strict_types=1);
+
+namespace Bazaarvoice\Connector\Plugin\ProductList;
+
+use Bazaarvoice\Connector\Model\Source\ProductList;
+
 /**
  * StoreFront Bazaarvoice Extension for Magento
```

Comment:
Should remove all references to "StoreFront"

Reply:
Fixed in other PR.